strings/base_coroutine_threadpool.h

```diff
@@ -21,16 +21,14 @@ namespace winrt::impl
         return (0 == WINRT_IMPL_CoGetApartmentType(&aptType, &aptTypeQualifier)) && ((aptType == 0 /*APTTYPE_STA*/) || (aptType == 3 /*APTTYPE_MAINSTA*/));
     }
 
-    inline bool requires_apartment_context() noexcept
+    inline auto current_context()
     {
-        int32_t aptType;
-        int32_t aptTypeQualifier;
-        return (0 == WINRT_IMPL_CoGetApartmentType(&aptType, &aptTypeQualifier)) && ((aptType == 0 /*APTTYPE_STA*/) || (aptType == 2 /*APTTYPE_NA*/) || (aptType == 3 /*APTTYPE_MAINSTA*/));
+        return capture<IContextCallback>(WINRT_IMPL_CoGetObjectContext);
     }
 
-    inline auto apartment_context()
+    inline auto is_default_mta_context(com_ptr<IContextCallback> const& context)
     {
-        return requires_apartment_context() ? capture<IContextCallback>(WINRT_IMPL_CoGetObjectContext) : nullptr;
+        return context == capture<IContextCallback>(WINRT_IMPL_CoGetDefaultContext, 1 /* APTTYPE_MTA */);
```

Comment:
AARGH! CoGetDefaultContext is not callable from UWP!

Reply:
Ignore my email. I now see why this is a problem. I filed [this task](https://microsoft.visualstudio.com/OS/_workitems/edit/26091860/) for you. How often will the WACK detect this? maybe it won't.

Reply:
The problem is more than WACK. It's the SDK. Windowsapp.lib doesn't contain CoGetDefaultContext. You'd need a new SDK to get a version that links.